src/main/kotlin/systems/ajax/englishstudytelegrambot/config/Configuration.kt

```diff
@@ -0,0 +1,20 @@
+package systems.ajax.englishstudytelegrambot.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.reactive.function.client.WebClient
+
+@Configuration
+class Configuration {
+
+    @Value("\${base.url.wordnikAPI}")
+    lateinit var baseUrlToWordnikAPI: String
+
+    @Bean
+    fun webClient(builder: WebClient.Builder): WebClient =
+        builder
+            .baseUrl(baseUrlToWordnikAPI)
+            .build()
+
+}
```

Comment:
```suggestion
            .build()
}
```

between last brace should be no empty lines. fix in all places